torch/_prims/context.py

```diff
@@ -87,6 +91,9 @@ def __torch_function__(
         mapping = torch_to_refs_map()
```

Comment:
This function seems to be only targetting torch.xxx.

In our dynamo workload, I think we are expecting an input GraphModule with `aten.ops`. Might want to expand this.

Reply:
Right, it's only torch.xxx for now. One option is to use dynamo under this context manager and another is to expand the context manager with "aten_to_refs_map", let's handle the extension in a separate PR.